qe/readme.md

```diff
@@ -0,0 +1,96 @@
+test suite. In this assignment, you'll build a small test suite, and tell us how
+you approached it.
+
+## What we're looking for
+
+Before we go into assignment requirements, here's what we're looking for in your
+submission, and how we grade it:
+
+- Are you able to build a test suite that tests provided scenarios?
+- Are you able to quickly come up with technology decisions (what framework to
+  use), scope decisions (what specifically to test, what edge cases are out of
+  scope, and why)?
+- Are you able to explain your decisions and your reasoning behind them,
+  clearly, in writing?
+- Is your code clean, readable, and maintainable?
+- Do you provide instructions on how to setup your test suite and run it? Does
+  it run reasonably well in our enginering environments?
+
+These are the questions that we'll have in mind when we look at your submission.
+
+## Requirements
+
+You'll build a test suite that tests a few scenarios on another SaaS that's
+built with Ruby on Rails — [Basecamp](https://basecamp.com/).
+
+To obtain an account to run tests against, please sign up for a new personal
+account on Basecamp, it's free.
+
+- **Build a test suite that runs the provided user scenarios**. We're
+  considering using Playwright, Cypress, or Selenium at Zipline, but we're
+  excited to see what framework you like most, and why.
+- **Provide instructions on how to run the suite from CLI**. The suite needs to
+  run on Linux, and Mac OS.
+- **You can use any libraries and dependencies you want**, as long as you
+  elaborate on why you chose to use them in the README of your submission.
+- If your suite needs any dependencies installed to run, **please provide
+  installation and usage instructions in the README**.
+- Provide a plan, and launch artifacts (`.circle.yaml` or github action spec?)
+  on how we should run the suite in production. We're using Github Actions, and
+  CircleCI, but feel free to use any runner you want, as long as we can hook
+  GitHub into it to trigger test runs. Pseudocode is fine.
+- Please provide a plan on how we'll be notified on test failures on this suite.
+  Suite does not have to report runs itself, if you decide to run it with GitHub
+  actions, for example.
+
+### User Scenarios to test
+
+Please build a suite that covers four scenarios:
+
+- **Successful login**: as a user, I can go to basecamp.com, navigate to the
+  login page, and provide my email and a valid password, and click sign in
+  button. I should be redirected to my Basecamp dashboard, and I should see my
+  Basecamp projects list that is not empty. In this scenario, it's fine to
+  create a test Basecamp project upfront.
+- **Unsuccessful login**: as a user, I can go to basecamp.com, navigate to the
+  sign in page, and fill in invalid email and password, click sign in button.
+  The login should fail, and I should see the login page again, with an error
+  message.
+- **Posting a message in a project**: TBD
```

Comment:
We need to add steps here for posting a message, let me know if that's something you want me to do?